src/shared/ui/components/BaseTable.vue

```diff
@@ -183,7 +190,7 @@ const pageSizeModel = computed({
 });
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
```

Comment:
Why?

Reply:
I think there were troubles with styles during development but now it doesn't affect anything so I will remove it.